svsim/src/main/scala/verilator/Backend.scala

```diff
@@ -12,6 +12,13 @@ object Backend {
     object TraceStyle {
       case class Vcd(traceUnderscore: Boolean = false, filename: String = "") extends TraceStyle
     }
+
+    sealed trait Timing
+    object Timing {
+      case object TimingEnabled extends Timing
+      case object TimingDisabled extends Timing
+      type Type = Timing
+    }
```

Comment:
Nit: I would do:

```suggestion
    object Timing {
      sealed trait Type
      case object TimingEnabled extends Type
      case object TimingDisabled extends Type
    }
```

Scala doesn't really have true enums, so enumerations are more of a pattern. This is usually the one I prefer.

Reply:
I see, noted. Thanks for clarifying this. I have updated it.